manifests/init.pp

```diff
@@ -111,7 +111,7 @@
     ensure  => present,
     source  => "${tmp_path}/composer.phar",
     require => [ Exec['download_composer'], File[$target_dir] ],
-    mode    => 0755,
+    mode    => '0755',
```

Comment:
It's already there in develop!!

Can you push it to master?
On Tue, Jan 27, 2015 at 11:02, "Alberto"  wrote:
In manifests/init.pp (https://github.com/enxebre/puppet-composer/pull/4#discussion_r23600493):
	> @@ -111,7 +111,7 @@ > ensure => present, > source => "${tmp_path}/composer.phar", > require => [ Exec['download_composer'], File[$target_dir] ], > - mode => 0755, > + mode => '0755',  Thanks, if you can create the pr for develop branch I'll merge it :)
	—
Reply to this email directly or view it on GitHub (https://github.com/enxebre/puppet-composer/pull/4/files#r23600493).